_minutes/2024-01-04-wecg.md

```diff
@@ -0,0 +1,158 @@
+ * [oliver] I like that too.
+ * [simeon] Con: that approach is that it takes two roundtrips. Pro: enables the ecosystem and allows us to pave cow paths in the future.
+ * [tomislav] Rob's proposal leans much in the API design that we already have.
+ * [oliver] For performance-sensitive data the storage API should probably not be used anyway.
+ * [rob] I'll raise my counter-proposal in the issue.
+
+[Issue 506](https://github.com/w3c/webextensions/issues/506): Firefox' webRequest.FilterResponse & more for all browsers
+
+ * [rob] The request here is basically implementing filterResponseData in Chrome/Firefox.
+ * [oliver] This definitely requires the blocking option.
+ * [rob] Does require it, but that's to ensure that the filter is registered in time. The feature request is the ability for an extension to put itself between the network response stream and the browser's processing of it.
+ * [oliver] I'm not sure if we would be willing to support that, because it would have the performance considerations that were relevant to dropping support for webRequestBlocking.
+ * [timothy] Yes, what Oliver said.
+ * [rob] With that, we could close the issue, because Chrome and Safari's positions are clear here.
+
+[Issue 507](https://github.com/w3c/webextensions/issues/507): Support $schema in manifest.json
+
+ * [patrick] The main request is to stop treating $schema as a hard error in Chrome. Other browsers don't fail here.
+ * [oliver] I think that we are supportive on Chrome's end - we are moving towards treating unrecognized/unsupported keys as non-fatal warnings instead of errors.
+ * [carlos] Currently you can load an extension just fine in Chrome with the $schema property. The issue seems to be present in the Chrome Web Store only.
+
+[Issue 510](https://github.com/w3c/webextensions/issues/510): Proposal: Increase maximum item size in Storage sync quotas
+
+ * [timothy] We don't currently support sync in Safari, but our preference is to keep things small.
+ * [rob] Devs shouldn't store too much data here. Breaking up the data into smaller pieces is a hack and against the spirit of the API limitations.
+ * [simeon] Sounds like this might be a documentation issue. We should clarify what the intent of this API is and that developers should use another system to sync user data.
+ * [rob] That having said, the current individual limit is 8k; do we want to keep it there or are we willing to raise it?
+ * [oliver] I'll check with the team.
+
+[Issue 511](https://github.com/w3c/webextensions/issues/511): Inconsistency: storage onChanged
+
+ * [tomislav] FYI on Firefox side we may not even be consistent across our namespaces. In some cases onChanged is fired when set() is called even without changes. I would not be opposed to changing it.
+ * [rob] For clarity, this issue is related to set() handling. Are all values propagated?
+ * [kiara] We only dispatch an update for changed values.
+ * [] How about sync, as mentioned in the issue?
```

Comment:
@Rob--W, @oliverdunk, @xeenon, any idea who said this?

Reply:
I don't know who said it, that's why I left it at `[]`. As this question was mainly a request for clarification, it's not an issue if the asker remains anonymous.

Reply:
I'm going to merge the PR before the next meeting (in 2 days). Any concerns before I do?